assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -739,3 +739,66 @@ form {
   padding: 0 $gu-h-spacing $gu-v-spacing;
   margin-top: $gu-v-spacing;
 }
+
+
+/* Server side rendering test loading message*/
+
+.form__radio-group--pills .amounts__placeholder .form__radio-group-input + .form__radio-group-label {
+  border: none;
+}
+
+.amounts__placeholder .form__radio-group-label {
```

Comment:
not for this pr but could this be perhaps a higher level body/html class? that way you could also serve the same html